package.json

```diff
@@ -9,7 +9,14 @@
     "test": "react-scripts-ts test --env=jsdom",
     "eject": "react-scripts-ts eject",
     "faker": "graphql-faker --extend http://localhost:8080/v5/graphql \\  -H 'Accept:application/json' -H 'Content-Type:application/json'",
-    "precommit": "check-dependencies && tslint --project . --type-check --fix"
+    "prettify": "./node_modules/.bin/prettier --write './src/**/*.{tsx,ts}'",
```

Comment:
You can omit the `./node_modules/.bin` part as it is already in the path [for npm scripts](https://docs.npmjs.com/misc/scripts#path)